lib/srv/desktop/rdp/rdpclient/client_common.go

```diff
@@ -61,7 +61,7 @@ type Config struct {
 // GenerateUserCertFn generates user certificates for RDP authentication.
 type GenerateUserCertFn func(ctx context.Context, username string, ttl time.Duration) (certDER, keyDER []byte, err error)
 
-//nolint:unused
+//nolint:unused // useful for debugging?
```

Comment:
Is it really unused? Could we delete it or tag the author? It looks like someone wrote it but forgot to add the calls.

Reply:
I've already fixed that. This function is used in a file that has a build flag.